pkg/jmxfetch/jmxfetch.go

```diff
@@ -217,6 +217,21 @@ func (j *JMXFetch) Start(manage bool) error {
 		return fmt.Errorf("incompatible options %q and %q", jvmContainerSupport, jvmCgroupMemoryAwareness)
 	} else if useContainerSupport {
 		javaOptions += jvmContainerSupport
+		maxHeapSizeAsPercentRAM := config.Datadog.GetFloat64("jmx_max_ram_percentage")
+		passOption := true
+		//These options overwrite the -XX:MaxRAMPercentage option, log a warning if they are found in the javaOptions
+		if strings.Contains(javaOptions, "Xmx") || strings.Contains(javaOptions, "XX:MaxHeapSize") {
+			log.Warnf("Java option -XX:MaxRAMPercentage will not take affect since at least one of the java options -Xmx and XX:MaxHeapSize, which both ovrerride MaxRAMPercentage, are present")
```

Comment:
```suggestion
			log.Warnf("Java option -XX:MaxRAMPercentage will not take effect since either -Xmx or XX:MaxHeapSize is already present. These options override MaxRAMPercentage.")
```